config/tgl-cavs.toml

```diff
@@ -227,3 +227,42 @@ count = 8
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 100000, 48, 48, 0, 1000, 0]
 
+	[[module.entry]]
+		name = "SRCINTC"
+		uuid = "e61bb28d-149a-4c1f-b709-46823ef5f5ae"
+		affinity_mask = "0xF"
+		instance_count = "10"
+		domain_types = "1"
+		load_type = "0"
+		module_type = "0x7"
+		auto_start = "0"
+		sched_caps = [1, 0x00008000]
+
+	# pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xffff, 0xc, 0x8, 0x45ff,
+		1, 0, 0xf6c9, 0xc, 0x8, 0x45ff]
+
+	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
+	mod_cfg = [0, 0, 0, 0, 12832, 1365500, 0, 0, 0, 1365, 0,
```

Comment:
Can the numbers be documented better? I have no idea what are the PAR_X and other acronyms.

Reply:
No document. It is copied from module spec and used for fw scheduler by windows. It is not supported by cSOF. use it for windows